lib/susedistribution.pm

```diff
@@ -683,7 +683,7 @@ sub activate_console {
     }
     elsif (
         $console eq 'installation'
-        && (((check_var('BACKEND', 's390x') || check_var('BACKEND', 'ipmi') || get_var('S390_ZKVM')))
+        && (((check_var('BACKEND', 's390x') || check_var('BACKEND', 'ipmi') || check_var('BACKEND', 'spvm') || get_var('S390_ZKVM')))
```

Comment:
@foursixnine afaics you are still working on the proper refactoring of remote backends right? This might be one case where we need to think about it, since this does not apply to svirt for non-s390x arch - which is not covered by `is_remote_backend` yet

Reply:
Are you saying that e.g. svirt-x86_64 would fail if we just replace all this backend checks with `is_remote_backend`?

Reply:
> Are you saying that e.g. svirt-x86_64 would fail if we just replace all this backend checks with `is_remote_backend`?

not 100% sure but the chances that it misbehaves are rather high imo.
@mnowaksuse wdyt?

Reply:
`svirt` does not have console of `ssh-xterm` type. So this is futile?

Reply:
I just saw this :(